src/main/java/cam72cam/immersiverailroading/tile/TileRailBase.java

```diff
@@ -109,6 +109,11 @@ public void setAugment(Augment augment) {
 		this.augment = augment;
 		if (getParentTile() != null) {
 			augmentGauge = getParentTile().info.settings.gauge;
+			if (augment == Augment.DETECTOR && ConfigDebug.defaultAugmentComputer) {
```

Comment:
I think you want an outer if statement for `ConfigDebug.defaultAugmentComputer`, with the existing if else inside of that.